pom.xml

```diff
@@ -5,7 +5,10 @@
     <groupId>com.pingcap.tispark</groupId>
```

Comment:
We didn't turn whole project into parent/child pom style. I wonder does this submodule and pom packaging works well for us? What's the behavior here?